pypsa/graph.py

```diff
@@ -51,16 +53,16 @@ def graph(
     """
     from pypsa import components
 
-    if isinstance(network, components.Network):
+    if isinstance(n, components.Network):
         if branch_components is None:
-            branch_components = network.branch_components
+            branch_components = n.branch_components
         else:
             branch_components = set(branch_components)
-        buses_i = network.buses.index
-    elif isinstance(network, components.SubNetwork):
+        buses_i = n.buses.index
+    elif isinstance(n, components.SubNetwork):
         if branch_components is None:
-            branch_components = network.network.passive_branch_components
-        buses_i = network.buses_i()
+            branch_components = n.n.passive_branch_components
```

Comment:
I am not sure whether subnetworks should rather have the attribute `network` instead of `n`.

Reply:
Hmm, I don't think so. All other related classes or accessors now use `self.n` instead of `network`, `_parent`, etc. Not sure why this should be different for `SubNetwork`?